docs/content/users/quickstart.md

```diff
@@ -47,6 +47,35 @@ ddev launch
     ddev launch
     ```
 
+## Contao
+
+=== "Composer"
+
+New Contao projects can be created from the official [managed edition](https://github.com/contao/managed-edition) using DDEV’s [`composer create` command](../users/usage/commands.md#composer):
+
+    ```shell
+    # Create a project directory and move into it:
+    mkdir my-contao-project && cd $_
```

Comment:
```suggestion
    mkdir my-contao-project
    cd my-contao-project
```
In regards of consistency i would stick to the pattern other quickstart guides use and write out the folder name on the change directory step, even it is longer